test/suites/smoke-test-common/test-configuration-consistency.ts

```diff
@@ -105,16 +111,25 @@ describeSuite({
 
                     // This should be true as long as they have enough credits for getting collators
                     for (const container of containersToCompareAgainst) {
-                        // we should only check those who have enough credits
-                        // we compare against the credits they had just before new session obviously
-                        // that is when they were charged for tokens
+                        // if not currently assigned, then one session
+                        // if currently assigned, then 2
+                        let sessionRequirements: bigint;
+
+                        if (
+                            currentAuthorityAssignment["containerChains"][container.toString()] == null ||
```

Comment:
Technically this should be undefined instead of null, but this also works because undefined == null returns true